### PR TITLE
Changes as per request in #5

### PR DESCRIPTION
**Pull request by BenVardy**
Change 'event/stats' to 'event/counts'
Store age grade as percent to 2d.p

### CODE REVIEW

File: design/database.d.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export interface event extends dbEntry {
     
     volunteerNames: runner[]
 
-    stats: {
+    counts: {
         no2k: number
```
Comment (ajdaniel) [inline design/database.d.ts]: As part of my request, could you also rename the properties of `count` to something a bit more readable, and do not include `no`? such as `2k` `5k` `runners` `volunteers`? The "number" part of the name is implied by being in the "count" property.

**Review by ajdaniel — CHANGES_REQUESTED** [review]
Just a minor change request please.

**Comment by markvardy** [comment]
@BenVardy Does the change from `stats` to `count` mean repopulating the database? If so, how can this be done for the existing data?

**Comment by ajdaniel** [comment]
@markvardy yes great point. We need all records to still match the same document design else the ui would fail when processing all documents for rendering

**Review by markvardy — APPROVED** [review]
I'm happy with this. We'll need to run the update script as soon as we've merged the changes, of course.

I assume you've tested the util script to update the db? (we need to talk about automated tests 😄 )

@ajdaniel you happy?

**Review by ajdaniel — APPROVED** [review]
I approve these changes. You could have saved some legwork by using the [ES6 object spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) operator like so:

```js
    docs = res.docs.map(doc => {
        if (doc.counts) {
            console.log(`'${doc._id}' already using 'count'`);
            return doc;
        }
        const { stats, ...restOfDoc } = doc;
        return {
            ...restOfDoc, // put the rest of the doc back without knowing the structure
            counts: {
                twok: stats.no2k,
                fivek: stats.no5k,
                total: stats.nototal,
                firstTimers: stats.noFirstTimes,
                pbs: stats.noPbs,
                volunteers: stats.noVolunteers
            },
        };
    });
```

**Pull request merged**